zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -344,6 +346,24 @@ public int progress() {
   }
 
   public boolean isBlankParagraph() {
+    // check interpreter-setting.json `config.checkEmpty` is equal false
+    Object configCheckEmpty = this.config.get(PARAGRAPH_CONFIG_CHECK_EMTPY);
+    if (null != configCheckEmpty) {
+      boolean checkEmtpy = PARAGRAPH_CONFIG_CHECK_EMTPY_DEFAULT;
+      try {
+        checkEmtpy = (boolean) configCheckEmpty;
+      } catch (ClassCastException e) {
+        LOGGER.error(e.getMessage(), e);
+      } catch (Exception e) {
+        LOGGER.error(e.getMessage(), e);
+      }
+      if (!checkEmtpy) {
+        LOGGER.info("This interpreter config `interpreter-setting.json` set config.{} = false", 
```

Comment:
i thought it's `interpreter.json `

Reply:
It's `interpreter-setting.json`.
e.g For example, paragraphs in submarine interpreter do not display title bar by default.
https://github.com/apache/zeppelin/blob/master/submarine/src/main/resources/interpreter-setting.json#L228